app/static/openapi.yaml

```diff
@@ -1,15 +1,13 @@
 openapi: 3.0.1
 info:
   title: ClimateMind API
-  description:
-    The API for sending and recieving information to and from the ClimateMind
-    ontology.
-  version: "0.1"
+  description: The API for sending and recieving information to and from the ClimateMind ontology.
+  version: '0.1'
 servers:
-  - url: http://127.0.0.1:5000
-  - url: http://localhost:5000
+  - url: 'http://127.0.0.1:5000'
+  - url: 'http://localhost:5000'
 paths:
-  /scores:
+  /alignment:
```

Comment:
This should remain /scores. It should not be /alignment. Right @danmash ?

Reply:
you are right